infer/src/integration/Driver.ml

```diff
@@ -60,6 +58,8 @@ let pp_mode fmt = function
       F.fprintf fmt "Gradle driver mode:@\nprog = '%s'@\nargs = %a" prog Pp.cli_args args
   | Javac {prog; args} ->
       F.fprintf fmt "Javac driver mode:@\nprog = '%s'@\nargs = %a" prog Pp.cli_args args
+  | JsonSIL {json_cfg; json_tenv} ->
+      F.fprintf fmt "Analyze json mode"
```

Comment:
typo: this isn't "analyze" anymore. Also it could be useful to dump the paths here to ease debugging, like the other modes.

Reply:
Good catch! Fixed